scripts/qa_crowbarsetup.sh

```diff
@@ -3931,6 +3931,10 @@ function oncontroller_testsetup
     . .openrc
     oncontroller_prepare_functional_tests
     if [[ $hacloud = 1 ]] ; then
+        # We expect the vip resources to have been disrupted the when the public network is created for the first time during the keystone
+        # proposal, so clean those up before checking for other failures.
+        crm_mon --failcounts -1 | grep "vip-admin-database-default-data:.*fail-count=1" && crm resource cleanup vip-admin-database-default-data
```

Comment:
That `-data` postfix is name of the cluster. See also https://github.com/SUSE-Cloud/automation/pull/1785